tools/tcpretrans.py

```diff
@@ -70,6 +74,24 @@
 };
 BPF_PERF_OUTPUT(ipv6_events);
 
+// separate flow keys per address family
+struct ipv4_fl_k_t {
+    // XXX: switch some to u32's when supported
+    u64 saddr;
+    u64 daddr;
+    u64 lport;
+    u64 dport;
+};
+BPF_HASH(ipv4_count, struct ipv4_fl_k_t);
+
+struct ipv6_fl_k_t {
+    unsigned __int128 saddr;
+    unsigned __int128 daddr;
+    u64 lport;
+    u64 dport;
+}__attribute__((packed));
+BPF_HASH(ipv6_count, struct ipv6_fl_k_t);
```

Comment:
There are no holes here, "__attribute__((packed))" is not needed, right?
Right now, we need to use "u64 lport" and "u64 dport". Once my struct/padding
patch landed, you can use type u16 instead of u64.

Reply:
ACK